docs/prms/oi1.rst

```diff
@@ -24,3 +24,12 @@ OI1 takes some optional arguments:
   * ``all``: connect the dummy node to all nodes in the interactome (i.e. full set of nodes in graph)
   * ``others``: connect the dummy node to all nodes that are not terminal nodes (i.e. nodes without prizes)
   * ``file``: connect the dummy node to a specific list of nodes provided in a file
+
+Dataset Usage
+-------------
+
+OmicsIntegrator1 prefers ``prize``s, but will take the union of ``sources`` and ``targets``
+and set their 'prize' to 1 if ``prize`` is not specified. If any ``dummy_nodes`` are specified,
+these are passed to OmicsIntegrator1 and can have their behavior configured with ``dummy_mode``.
+
+OmicsIntegrator1 considers graph directionality.
```

Comment:
should the directionality implementation be explained here as well? Essentially the algortihm takes in the interactome as is becuase it can deal with directionality?